webui/react/src/hooks/useModal/JupyterLab/useModalJupyterLab.tsx

```diff
@@ -279,19 +284,24 @@ const JupyterLabFullConfig: React.FC<FullConfigProps> = ({
               wrappingIndent: 'indent',
             }}
           />
-        </Item>
+        </Form.Item>
       </React.Suspense>
       {configError && <Alert message={configError} type="error" />}
     </Form>
   );
 };
 
-const JupyterLabForm: React.FC<FormProps> = ({ updateFields, fields }: FormProps) => {
+const JupyterLabForm: React.FC<{
+  defaults: JupyterLabOptions;
+  form: FormInstance<JupyterLabOptions>;
+}> = ({ form, defaults }) => {
   const [templates, setTemplates] = useState<Template[]>([]);
   const [resourcePools, setResourcePools] = useState<ResourcePool[]>([]);
```

Comment:
Consider migrating fetching the resource pool list to using the model introduced in this recent PR https://github.com/determined-ai/determined/pull/5543